From e1da1d6ab6c077b0ff64aaef913fe76c647493e3 Mon Sep 17 00:00:00 2001
From: marco-m <marco.molteni@laposte.net>
Date: Fri, 28 Feb 2014 15:21:39 +0100
Subject: [PATCH] Fix typo.

---
 src/Wt/Auth/PasswordService |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Wt/Auth/PasswordService b/src/Wt/Auth/PasswordService
index 4d76f89..af7ee7b 100644
--- a/src/Wt/Auth/PasswordService
+++ b/src/Wt/Auth/PasswordService
@@ -117,9 +117,9 @@ public:
    */
   AbstractVerifier *verifier() const { return verifier_; }
 
-  /*! \brief Sets a validator which computes password strenght.
+  /*! \brief Sets a validator which computes password strength.
    *
-   * The default password strenght validator is \c 0.
+   * The default password strength validator is \c 0.
    *
    * The service takes ownership of the validator.
    */
-- 
1.5.4.6

